@here/harp-datasource-protocol/lib/Expr.ts

```diff
@@ -195,31 +197,34 @@ interface ReferenceResolverState {
 }
 
 /**
- * The evaluation scope of an [[Expr]].
+ * The evaluation scope of an {@link Expr}.
  */
 export enum ExprScope {
     /**
-     * The scope of an [[Expr]] used as value of an attribute.
+     * The scope of an {@link Expr} used as value of an attribute.
      */
     Value,
 
     /**
-     * The scope of an [[Expr]] used in a [[Technique]] `when` condition.
+     * The scope of an {@link Expr} used in a [[Technique]] `when` condition.
      */
     Condition,
 
     /**
-     * The scope of an [[Expr]] used as dynamic property attribute value.
+     * The scope of an {@link Expr} used as dynamic property attribute value.
      */
     Dynamic
 }
 
+// tslint:disable:max-line-length
```

Comment:
this disables it for the rest of the file, is that what you want?

Reply:
oops, how can i disable the max-line-length only for the comment?